core/client/pom.xml

```diff
@@ -188,6 +188,12 @@
           <scope>provided</scope>
         </dependency>
         <dependency>
+          <groupId>com.google.inject</groupId>
+          <artifactId>guice</artifactId>
+          <version>3.0</version>
```

Comment:
Ideally I want to exclude the dependency, regardless of the version. However, maven complains and requires the version to be provided.

I'm reluctant to put in dependency management, because I treat it as the central place for all *direct* dependencies that Alluxio pulls in, and therefore lock the version down.

I feel here is a bit special, because it's pertaining only to a particular profile, and Alluxio doesn't care its version, because there's code dependency on it. Does this make sense?